flashpay/apps/payments/tests/test_views.py

```diff
@@ -241,6 +305,7 @@ def test_verify_transaction_with_wrong_txn_hash_and_txn_note(
         short_name="CHOICE",
         long_name="Choice Coin",
         image_url="https://hi.com/choice-coin",
+        decimal=2,
     )
```

Comment:
why `choice`?  I know it's for testing but I'd suggest you use `USDT`, `USDC`,  `ALGO` as those are our supported assets

Also, you should write tests for;
1. invalid amounts
2. not opted conditions
3. invalid address